docs/detections/rules-ui-create.asciidoc

```diff
@@ -114,75 +125,137 @@ image::images/rule-query-example.png[]
+
+* To create an event correlation rule using Event Query Language, select *Event Correlation* and then:
+.. Define which {es} indices the rule searches for alerts.
+.. Add an {ref}/eql-syntax.html[EQL statement] used to detect alerts.
+
+For example, the following rule detects when `msxsl.exe` makes an outbound
+network connection:
+
+** *Index patterns*: `winlogbeat-*`
++
+*Defintion*: Winlogbeat ships Windows events to {es-sec}.
+** *EQL query*:
++
+[source,eql]
+----
+sequence by process.entity_id
+  [process
+    where event.type in ("start", "process_started")
+    and process.name == "msxsl.exe"]
+  [network
+    where event.type == "connection"
+    and process.name == "msxsl.exe"
+    and network.direction == "outgoing"`
+----
++
+*Definition*: Searches the `winlogbeat-*` indices for sequences of a `msxsl.exe` process start
+event followed by an outbound network connection event that was started by the
+`msxsl.exe` process.
+
+[role="screenshot"]
+image::images/eql-rule-query-example.png[]
 
-** _Custom query_: `host.name : liv-win-19 and event.category : "authentication" and event.outcome : "failure"`
-** _Field_: Leave blank
-** _Threshold_: `10`
+NOTE: For sequence events, the {security-app} generates a single alert when all events listed in the sequence are detected. To see the matched sequence events in more detail, you can view the alert in the Timeline, and, if all events came from the same process, open the alert in Analyze Event view.
 
+* To create a threat match rule that searches the specified threat indices for
+field values, select *Threat Match* and then fill in these fields:
+.. *Index patterns*: The {es-sec} event indices on which the rules runs.
+.. *Custom query*: The query and filters used to retrieve the required results from
+the {es-sec} event indices. For example, if you only need to check
+`destination.ip` event values, add `destination.ip : *`.
++
+TIP: If you want the rule to check every field in the indices, use this
+wildcard expression: `*:*`.
+.. *Threat index patterns*: The threat indices containing field values for which
+you want to generate alerts.
+.. *Threat index query*: The query and filters used to filter the fields from
+the threat indices.
+.. *Threat Mapping*: Compares the values of the specified event and threat field
+values. When the field values are identical, an alert is generated. To define
+which field values are compared from the indices:
+** *Field*: The field used for comparing values in the {es-sec} event
+indices.
+** *Threat index field*: The field used for comparing values in the threat
+indices.
+.. You can add `AND` and `OR` clauses to define when alerts are generated.
++
+For example, to create a rule that generates alerts when `host.name` *and*
+`destination.ip` field values in the `logs-*` or `packetbeat-*` {es-sec} indices
+are identical to the corresponding field values in the `ip-threat-list` threat
+index, enter the rule parameters seen in the following image:
++
```

Comment:
Relaying a suggestion from @aarju here. 
> It would also help a lot if there was an example of how to format an index to be used as a threat index. For example, if I have a list of 10 domains and their IP addresses that I want to monitor, how do I add those to Elastic to be used as a thread index

Reply:
@peluja1012 I think this is a good idea, but I probably won't have time to incorporate it before tomorrow unless it's a simple process. This might be best as a separate topic since this one is already pretty long as is. WDYT?

Reply:
I think it makes sense to include it a separate topic. We could link to it from this page. 👍